### ISSUE TITLE
Stuff to explore from the top submission

### ISSUE THREAD

**Issue by jamesqo** [jamesqo/housing-project#1]
- Using skewness / kurtosis to decide how features should be transformed
- Using matplotlib to generate plots easily
- Boxcox transformation
- Robust scaler
- Power transformation
- Gradient boosting (xgboost, lightgbm, sklearn's `GradientBoostingRegressor`)
- `MLPRegressor`
- Various classes to help with cross-validation
- DBSCAN clustering
- Investigate how clustering algorithms like nearest neighbors, PCA were used to detect outliers
